plugins/CoreVisualizations/Visualizations/Cloud.php

```diff
@@ -50,7 +50,7 @@ public function beforeLoadDataTable()
         $this->checkRequestIsNotForMultiplePeriods();
     }
 
-    public function afterAllFiltersAreApplied()
+    public function beforeGenericFiltersAreAppliedToLoadedDataTable()
```

Comment:
This might possibly break cloud viz for processed metrics. Processed metrics are calculated within the generic filters, so if the cloud is meant to be shown for a processed metric this might possibly fail then.

But there might actually be a much more reliable solution for this. The cloud viz doesn't show any metric values at all. So it should be possible to simply fetch all data unformatted (format_metrics=0).

Guess something like this in the `beforeLoadDataTable` could work:
```
$this->requestConfig->request_parameters_to_modify['format_metrics'] = 0;
```

Reply:
Thanks @sgiehl, that's a cleaner fix. I've updated the PR to use this approach instead of changing the method and it's working well :+1: